HelpSource/Classes/QuarksGui.schelp

```diff
@@ -0,0 +1,23 @@
+class:: QuarksGui
+summary:: Allows Quarks Installation
+related:: Classes/Quarks
+
+Description::
+
+Quarks are packages of SuperCollider code containing classes, extension methods, documentation and server UGen plugins. The QuarksGui class acts as a frontend to Quarks, being instanted in the Quarks Class, to call its UI/Frontend, just as you can see if you open the its respective *.sc  file. The created UI/Fronted window allows to manage downloading these packages and installing or uninstalling them.
```

Comment:
Users generally don't use this class directly. IMO the only documentation required is to state that users create a QuarksGui by running `Quarks.gui`, and link to the main Quarks documentation. Users are not expected to call any methods on this class directly, so we should not waste effort documenting what they aren't supposed to use anyway.